src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -39,6 +55,60 @@ export class CreateClusterComponent {
         NotificationType.info,
         $localize`Cluster creation skipped by user`
       );
+      this.router.navigate(['/dashboard']);
     });
   }
+
+  onSubmit() {
+    forkJoin(this.observables)
+      .pipe(
+        finalize(() =>
+          this.clusterService.updateStatus('POST_INSTALLED').subscribe(() => {
+            this.notificationService.show(
+              NotificationType.success,
+              $localize`Cluster creation was successful`
+            );
+            this.router.navigate(['dashboard']);
```

Comment:
Shouldn't it be  `this.router.navigate(['/dashboard']);` ?

Reply:
Done.